docs/standard-library/http/content-types.md

```diff
@@ -74,26 +69,3 @@ op getAvatarAsPng(@header accept: "image/png"): PngImage;
 @sharedRoute
 op getAvatarAsJpeg(@header accept: "image/jpeg"): JpegImage;
 ```
-
-### Option 2: Using overload
```

Comment:
Not sure why we are removing this - it should be possible for folks to use this mechanism, even though it is not supported in DPG emitters

Reply:
yeah lets discuss that in the dpg meeting, maybe what we want is a linter rule for azure spec not to use `@overload` or we can discusss if we want to stop supporting it all together but I don't think we should just remove it from the public doc. Azure docs should already have similar docs with the resource templates that only show the sharedRoute flow